File: app/src/main/java/com/example/esl/models/local/entities/Notifikasi.kt
package com.example.esl.models.local.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Notifikasi")
data class Notifikasi(
    @PrimaryKey(autoGenerate = true) val id_notifikasi: Int = 0,
    val id_users: Int,
    val tanggal_kirim: String,
    val status_notifikasi: String // Enum: unread, read)
)

File: models/notifikasi.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Notifikasi extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      Notifikasi.belongsTo(models.Users, { foreignKey: 'id_users' });
    }
  }
  Notifikasi.init({
    id_notifikasi: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true,
    },
    id_users: DataTypes.INTEGER,
    tanggal_kirim: DataTypes.DATE,
    status_notifikasi: {
      type: DataTypes.ENUM,
    values: ["read", "unread"],
  }
}, {
    sequelize,
    modelName: 'Notifikasi',
  });
  return Notifikasi;
};